Sources/TokamakCore/Views/Button.swift

```diff
@@ -15,6 +15,25 @@
 //  Created by Max Desiatov on 02/12/2018.
 //
 
+/// A control that performs an action when triggered.
+///
+/// Available when `Label` conforms to `View`.
+///
+/// A button is created using a `Label` an action (a method or closure) to be called on click.
```

Comment:
This doesn't read smoothly to me, how's about slightly changing it similar to this?
```suggestion
/// A button is created using a `Label`, the `action` initializer argument (a method or closure) 
/// is called on click.
```